src/index.html

```diff
@@ -8,6 +8,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+
+    <div class="stars stars--1">
+      <div class="stars__star star--active"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+
+    <div class="stars stars--2">
+      <div class="stars__star star--active"></div>
+      <div class="stars__star star--active"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+
+    <div class="stars stars--3">
+      <div class="stars__star star--active"></div>
+      <div class="stars__star star--active"></div>
+      <div class="stars__star star--active"></div>
```

Comment:
It is cheating)
You do not need to create an active class for specific `stars__star` items. In the conditions of this task, they should be highlighted under the influence of the parent modifier such as `stars--3`

Reply:
Is this solution better?)
![4](https://github.com/mate-academy/layout_stars/assets/91736996/92d01264-c6bf-4bcd-915c-f380f4f8f014)